test/integration/kafka_sarama/kafka_test.go

```diff
@@ -30,7 +33,16 @@ func TestSendEvent(t *testing.T) {
 		clienttest.SendReceive(t, func() interface{} {
 			return protocolFactory(t)
 		}, eventIn, func(e event.Event) {
-			test.AssertEventEquals(t, eventIn, test.ConvertEventExtensionsToString(t, e))
+			got := test.ConvertEventExtensionsToString(t, e)
+			extensions := got.Extensions()
+			require.NotEmpty(t, extensions[KAFKA_OFFSET])
+			require.NotEmpty(t, extensions[KAFKA_PARTITION])
+			require.NotEmpty(t, extensions[KAFKA_TOPIC])
```

Comment:
can we check their values? Should they match the constants defined above?

Reply:
Sure! But we can only verify the topic defined above. And the partition/offset isn't defined here.